src/components/MyKiva/MyKivaEmailUpdatesCard.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+	<div class="card-container tw-h-full">
+		<span
+			class="tw-inline-flex tw-items-center tw-gap-1 tw-mb-2
+						tw-rounded-md tw-bg-eco-green-1 tw-px-1.5 tw-py-0.5
+						tw-absolute tw-top-2.5 tw-left-2.5 tw-z-1"
+		>
+			<KvMaterialIcon
+				class="tw-w-2 tw-h-2 tw-shrink-0"
+				:icon="mdiEmailOutline"
+			/>
+			<span class="tw-text-primary tw-font-medium tw-align-middle" style="font-size: 0.875rem;">
+				Email updates
+			</span>
+		</span>
+		<div class="tw-relative tw-w-full tw-overflow-hidden tw-rounded image-container">
+			<KvBorrowerImage
+				class="tw-w-full tw-h-full tw-object-cover"
+				:alt="name"
+				:hash="hash"
+				:aspect-ratio="3 / 4"
+				:default-image="{ width: 304 }"
+				:images="[{ width: 304 }]"
+				:photo-path="$appConfig.photoPath"
+			/>
+		</div>
+		<div class="tw-flex tw-flex-col tw-pt-1">
```

Comment:
This also needs `tw-grow`. These two class changes ensure the bottom content expands to the height of the parent box. You'll see in storybook that there is extra space at the bottom without these two small changes.

<img width="418" height="401" alt="Image" src="https://github.com/user-attachments/assets/645299b1-d37b-48bf-8ecc-1543c5c3d65f" />